src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -62,6 +62,14 @@ def make_artifacts(
 
 @click.command()
 @click.argument("output_dir", type=click.Path(exists=True))
+@click.option(
+    "--version",
+    type=str,
+    default=None,
+    show_default=True,
+    help="Provide a version number for final results. If None is provided, a datetime "
```

Comment:
Oh, probably want to specify in this help string the format of version, e.g.
```
help="Provide a version number for final results. Must be formatted like '#.#.#'. If None is provided..."
```

And then throw an error if they provide a different version formatting.